src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -28,10 +32,40 @@ export interface DisplayAdContainerProps extends FlexProps {
 GPT.syncCorrelator(true)
 
 export const DisplayAd: SFC<DisplayAdProps> = props => {
-  const { adDimension, adUnit, targetingData, ...otherProps } = props
+  const {
+    adDimension,
+    adUnit,
+    targetingData,
+    articleSlug,
+    ...otherProps
+  } = props
   const [width, height] = adDimension.split("x").map(a => parseInt(a))
   const [isAdEmpty, setAdEmpty] = useState(null)
   const isMobileLeaderboardAd = is300x50AdUnit(adDimension)
+  const { trackEvent } = useTracking()
+  const isMounted = useDidMount()
+
+  if (isMounted) {
+    trackEvent({
```

Comment:
Hmm, yes, and actually I wonder if the hook should be updated to handle this since I would assume it's intended to only be called once after the component mounts as well.  I'll look into it and make the changes.

Reply:
I guess I thought that's what `useDidMount` was doing as well because it looks like it's just a wrapper for `useEffect` which receives the empty array to only fire once.

Reply:
Maybe it's worth checking in with @kierangillen to see what the use case for `useDidMount` is but I feel like both ways of writing this are correct.

Reply:
The way I read the code for `useDidMount`, it is firing an effect once, but that effect is to update a state property that tracks whether the component has "mounted" or not. This would be useful for a situation where you didn't want something to happen in your component until you were sure the component had mounted, but it doesn't do anything to restrict things from happening every re-render after it's mounted. Our scenario is different - we want something to happen once the component is first rendered, *but never again*.